Editor/HPUIInteractorConeRayAnglesEditor.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEditor;
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine.XR.Hands;
+
+namespace ubco.ovilab.HPUI.Editor
+{
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(HPUIInteractorConeRayAngles), true)]
+    public class HPUIInteractorConeAnglesEditor: UnityEditor.Editor
+    {
+        private HPUIInteractorConeRayAngles t;
+
+        private float offsetValue;
+        private bool applyToAll;
+        private bool overwriteThisAsset;
+        private XRHandJointID selectedPhalange = XRHandJointID.IndexDistal;
+
+        // storing all baseline thresholds
+        private Dictionary<XRHandJointID, List<float>> baselineThresholds = new Dictionary<XRHandJointID, List<float>>();
+
+        private const string DONT_ASK_FOR_OVERRIDE_EDITOR_PREF_KEY = "ubco.ovilab.HPUI.Interaction.Components.ConeThresholdsEditor.DontAskForOverride";
+        private bool userSelection;
+        private string saveName;
+
+        private bool advancedSettingsFoldout;
+
+        protected void OnEnable()
+        {
+            t = target as HPUIInteractorConeRayAngles;
+            if (!t) return;
+            foreach ((XRHandJointID phalange, List<HPUIInteractorRayAngle> rays) in t.ActiveFingerAngles)
+            {
+                if (!baselineThresholds.ContainsKey(phalange) && rays.Count > 0)
+                {
+                    baselineThresholds[phalange] = rays.Select(ray => ray.RaySelectionThreshold).ToList();
+                }
+            }
+        }
+
+        public override void OnInspectorGUI()
+        {
+            base.OnInspectorGUI();
+
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+
+            advancedSettingsFoldout = EditorGUILayout.Foldout(advancedSettingsFoldout, "Advanced Settings", toggleOnLabelClick:true);
```

Comment:
let's call this advanced operations or something. I don't want this to be called settings.